src/coreclr/jit/importercalls.cpp

```diff
@@ -1844,18 +1844,18 @@ GenTreeCall* Compiler::impImportIndirectCall(CORINFO_SIG_INFO* sig, const DebugI
 // Arguments:
 //    call - The unmanaged call
 //    sig  - The signature of the call site
-//    swiftErrorArg - [out] If this is a Swift call with a SwiftError* argument, then the argument is returned here.
-//                    Otherwise left at its existing value.
 //
-void Compiler::impPopArgsForUnmanagedCall(GenTreeCall* call, CORINFO_SIG_INFO* sig, CallArg** swiftErrorArg)
+// Return Value:
+//    Pointer to SwiftError* arg's GenTree node, if call is a Swift call with error handling
+//
+GenTree* Compiler::impPopArgsForUnmanagedCall(GenTreeCall* call, CORINFO_SIG_INFO* sig)
```

Comment:
It feels weird to make this the sole return value of the function. I would keep it as an out parameter.

Reply:
I just reread my feedback and I guess it was confusing phrasing from my side, sorry about that.